packages/react/src/primitives/shared/constants.ts

```diff
@@ -958,6 +958,8 @@ export const ComponentText = {
 
 export const stylePropsToThemeKeys: Record<ThemeStylePropKey, ThemeTokenKey> = {
   backgroundColor: 'colors',
+  borderColor: 'colors',
+  borderWidth: 'space',
```

Comment:
```suggestion
  borderWidth: 'borderWidths',
```
This will let it utilize the available borderWidths tokens defined in the theme.